errors/names.go

```diff
@@ -21,23 +21,23 @@ import (
 const (
 	// RetryError occurs when checks fail after exhausting all retry attempts.
```

Comment:
these changes are irrelevant and will disappear. There are remnants of a previous PR.